osgar/record.py

```diff
@@ -31,7 +31,12 @@ def __init__(self, config, logger, application=None):
                 assert application is not None  # external application required
                 module = application(module_config['init'], bus=bus)
             else:
-                module = all_drivers[module_class](module_config['init'], bus=bus)
+                try:
+                    driver = all_drivers[module_class]
```

Comment:
Can we ditch `all_drivers` now and handle everything with `importlib`? `get_class_by_name` would import from drivers package if there is no '.' inside the name.

Reply:
OK, I moved that into `config.py` but I still use `all_drivers` for mapping names to classes. We may even abandon keyword "application", but then we will need specific config for each application.

Reply:
Thanks, that is better. I was thinking even about removing `all_drivers` completely and using `importlib` for them as well. Something like:

```python
m = import_module("." + name.lower(), 'osgar.drivers')
klass = getattr(m, name)
```

But feel free to keep it as you have it now if you like it better.

Reply:
ok